packages/dapp-svelte-wallet/ui/src/Balance.svelte

```diff
@@ -0,0 +1,41 @@
+<script>
+    import Purse from './Purse.svelte';
+
+    export let item;
+    const icons = {
```

Comment:
This should work for the demo, but I'm not sure the best way going forward to get icons for purses since petnames can change.